docs/source/guides/configuration.rst

```diff
@@ -101,6 +102,8 @@ Environment-specific parameters
   This parameter allows users to find an optimal value in a tradeoff between
   resource consumption and the delay of getting back to work after an idle
   period.
+- ``mst_stalled_batch_threshold`` is an optional parameter that sets the time threshold after which inactive batches are considered not present on other peers (in milliseconds).
```

Comment:
Let's bind this abstract statement to user's needs:
- if a node was restarted, it will recover MST state after approx this period
- in case of network failures, same
- if too small, odd traffic is generated and processor gets busy
- if too large, recovery from MST problems delays